### DISCUSSION TITLE
Never name your python file "photoshop.py"

### DISCUSSION THREAD

**Discussion by muhammad-mobeen** [loonghao/photoshop-python-api#190] [General]
I just pip-installed this API and was testing it with the hello world template but it was giving me the following error:-

"No module named 'photoshop.api'; 'photoshop' is not a package"

I wasted countless hours to find the solution with no luck and then a eureka moment happened I changed the file name from "photoshop.py" to "ps.py" and it worked like a charm. The problem was probably because the file name confused python on what to import. So silly.

**Comment by loonghao** [comment]
@muhammad-mobeen 

Did you use pip to install this package?
```
pip install photoshop-python-api
```

**Reply by muhammad-mobeen** [reply]
Yes, absolutely I did.

**Comment by loonghao** [comment]
@muhammad-mobeen Could please give me more context information? such as your python version, and your system version, we tested and support windows 7 or windows 10.
